prime-dotnet-webapi/PrimeEnvironment.cs

```diff
@@ -70,5 +70,12 @@ public static class AddressAutocompleteApi
             public readonly static string Url = Environment.GetEnvironmentVariable("ADDRESS_AUTOCOMPLETE_API_URL") ?? "https://ws1.postescanada-canadapost.ca/AddressComplete/Interactive/";
             public readonly static string Key = Environment.GetEnvironmentVariable("ADDRESS_AUTOCOMPLETE_API_KEY") ?? "XC95-NM62-BR64-TG37";
         }
+
+        public static class MetabaseApi
+        {
+            public readonly static string Url = Environment.GetEnvironmentVariable("METABASE_SITE_URL") ?? "https://metabase-test.pharmanetenrolment.gov.bc.ca";
+            public readonly static string Key = Environment.GetEnvironmentVariable("METABASE_SECRET_KEY") ?? "f7a2ffbc8ebd7e273603896c0f63ae04a596b94ff76c276398ce5aa8ca216cee";
+            public readonly static string DashboardId = Environment.GetEnvironmentVariable("METABASE_DASHBOARD_ID") ?? "4";
```

Comment:
A bit cleaner to parse this here rather than in the service. Take a look at MailServer.Port for an example.